components/proxy/conf/Caddyfile

```diff
@@ -8,7 +8,7 @@
 	# set default SNI for old clients
 	default_sni {$GITPOD_DOMAIN}
 
-	# debug
+	debug
```

Comment:
We should probably remove/comment this out again.

Reply:
Done.